tests/sles4sap/hana_install.pm

```diff
@@ -45,6 +45,18 @@ sub download_hana_assets_from_server {
     # Each HANA asset is about 16GB. A ten minute timeout assumes a generous
     # 27.3MB/s download speed. Adjust according to expected server conditions.
     assert_script_run "wget -O - $hana_location | tar -xf -", timeout => $nettout;
+    # Skip checksum check if DISABLE_CHECKSUM is set, or if checksum file is not
+    # part of the archive
+    my $sap_chksum_file = 'MD5FILE.DAT';
+    my $chksum_file = 'checksum.md5sum';
+    my $no_checksum_file = script_run "[[ -f $target/$chksum_file || -f $target/$sap_chksum_file ]]";
+    return 1 if (get_var('DISABLE_CHECKSUM') || $no_checksum_file);
+
+    # Switch to $target to verify copied contents are OK
+    assert_script_run "pushd $target";
```

Comment:
Since L60 is doing `assert_script_run "cd";`, I think it would be better to use `cd` here too, or add a `popd` below after the check.

Reply:
`pushd $target; ... ; ... ; cd.` is something what's already there, I just add the line with `md5sum`, unfortunately  github don't show ups side by side diffs